src/graphql/schema.prisma

```diff
@@ -14,6 +14,7 @@ model User {
     id                  Int                  @id @default(autoincrement())
     name                String
     short               String               @unique
```

Comment:
We should probably rename this to `username` 🤔

Reply:
yeah. Might cause confusion otherwise